### PR TITLE
Add chatterbox badge

### PR DESCRIPTION
**Pull request by ErikSchierboom**
- Fix invalid test file names
- Add chatterbox badge

### CODE REVIEW

File: app/images/icons/chatterbox.svg
Comment (ErikSchierboom) [inline app/images/icons/chatterbox.svg:1]: TODO

File: app/models/badges/chatterbox_badge.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+module Badges
+  class ChatterboxBadge < Badge
+    seed "Chatterbox",
+      :common,
+      :chatterbox,
+      "Linked your Exercism account to Discord"
+
+    def award_to?(user) = user.discord_uid.present?
+
+    def send_email_on_acquisition? = false
```
Comment (ErikSchierboom) [inline app/models/badges/chatterbox_badge.rb]: Not sure if we wanted to send an email

File: app/models/badges/chatterbox_badge.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+module Badges
+  class ChatterboxBadge < Badge
+    seed "Chatterbox",
+      :common,
```
Comment (ErikSchierboom) [inline app/models/badges/chatterbox_badge.rb:4]: I was unsure about the rarity

Comment (iHiD) [reply app/models/badges/chatterbox_badge.rb:4]: common is good.

File: app/models/badges/chatterbox_badge.rb (hunk 1 above)
Comment (iHiD) [reply app/models/badges/chatterbox_badge.rb]: I'd say `false` in general, but I'd like to email it when we batch award it. So maybe deploy it as true, do the batch, then change it to false. Same for the forum one.

File: app/models/badges/chatterbox_badge.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+      :chatterbox,
+      "Linked your Exercism account to Discord"
```
Comment (iHiD) [inline app/models/badges/chatterbox_badge.rb]:
```suggestion
      "Joined Exercism's Discord server"
```

**Review by iHiD — APPROVED** [review]

**Pull request merged**